ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -904,3 +904,101 @@ def test_ipahealthcheck_trust_agent_member_check(self):
         for check in data:
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
+
+
+class TestIpaHealthCheckWithExternalSelfSignedCA(IntegrationTest):
+    """
+    Tests to run and check whether ipa-healthcheck tool reports correct status when
+    IPA Master has external self signed CA.
+    """
+
+    @classmethod
+    def install(cls, mh):
+        result = install_server_external_ca_step1(cls.master)
```

Comment:
You need to import this method from `test_integration/test_external_ca.py`.
OR
You can move these method (`test_external_ca.install_server_external_ca_step1()`) to `tasks.py`

Reply:
missing import statement was added.